classes/Provider/TalkHelperProvider.php

```diff
@@ -22,10 +22,48 @@
     public function register(Container $app)
     {
         $app[TalkHelper::class] = function ($app) {
+            $categories = $app->config('talk.categories');
+
+            if ($categories === null) {
+                $categories = [
```

Comment:
I assume this is for displaying default values in case they aren't set in the config file.

Reply:
Yes, it is, but why would they not be set?

Reply:
When it was introduced in this commit: https://github.com/opencfp/opencfp/commit/f3b9384b943b8e83177c05a765648a19be3253c1, i assume it was so people would have an easier time upgrading.

By now i think it should throw an error if it is not properly set.This would make sure people wouldn't launch their CFP with the wrong categories/types etc

Reply:
This raises a couple of questions:

* Do people upgrade?
* If so, how?

Maybe we should also consider making this application available via packagist, so people can run

```
$ composer composer create-project opencfp/opencfp cfp.foobar.com
```

?

/cc @chartjes @mdwheele